tests/virt_autotest/virt_autotest_base.pm

```diff
@@ -257,7 +257,15 @@ sub post_fail_hook {
     $self->post_run_test;
     save_screenshot;
 
-    if (get_var('VIRT_PRJ2_HOST_UPGRADE')) {
+    if (get_var('VIRT_PRJ1_GUEST_INSTALL')) {
+        #collect and upload guest autoyast control files
+        my $guest_autoyast_dir = "/guest_autoyast_files";
+        my $guest_autoyast_src = "/srv/www/htdocs/install/autoyast/";
+        assert_script_run "mkdir -p $guest_autoyast_dir; cp $guest_autoyast_src* $guest_autoyast_dir";
```

Comment:
Could you help explain why not directly pass /srv/www/htdocs/install/autoyast/ to virt_utils::collect_host_and_guest_logs ?

Reply:
yep,  if directly copy guest autoyast file from /srv/www/htdocs/install/autoyast/ to our log_root dir here via virt_utils::collect_host_and_guest_logs.
will changed our log directory as below:
```
log_root
    \__host_xx_extra_logs
           \__/srv/www/htdocs/install/autoyast
                               \___sles-15-sp2-64-fv-def-net.autoyast 
```
/srv/www/htdocs/install/autoyast under our log directory, will not make sense here, imho.                       

Also, refer to Alice's comment about our log directory as below
```
log_root
    \__host_xx_extra_logs
           \__guest_autoyast_files
                    \___sles-15-sp2-64-fv-def-net.autoyast 
```
So, do not directly copy guest autoyast file from /srv/www/htdocs/install/autoyast/ to our log_root dir here.

Reply:
@waynechen55 hope all comments were solved now. please help me make sure if there was new comment for this PR.
if not, this PR is available to merge now. Thanks.

Reply:
I think this part can be further simplified as:

assert_script_run "cp -r /srv/www/htdocs/install/autoyast .; mv autoyast guest_autoyast_files".

Reply:
I think this part can be further simplified as:

assert_script_run "cp -r /srv/www/htdocs/install/autoyast .; mv autoyast guest_autoyast_files".

Reply:
Then        
virt_utils::collect_host_and_guest_logs('', '/guest_autoyast_files', '');
assert_script_run "rm -rf /guest_autoyast_files";

Reply:
Cool! had updated as bellow:
```
        assert_script_run "cp -r /srv/www/htdocs/install/autoyast /guest_autoyast_files";
        virt_utils::collect_host_and_guest_logs('', '/guest_autoyast_files', '');
        assert_script_run "rm -rf /guest_autoyast_files";
```

Reply:
LGTM

Reply:
Add the latest validation run:
https://openqa.suse.de/tests/5813116/file/autoinst-log.txt
//
host_ix64ph1087_extra_logs/guest_autoyast_files/
host_ix64ph1087_extra_logs/guest_autoyast_files/sles-11-sp4-64-fv-def-net
host_ix64ph1087_extra_logs/guest_autoyast_files/sles-11-sp4-64-pv-def-net
...
//
@alice-suse this PR is ready to merge now.